### PR TITLE
TBRANDS-114 - Image Updates & Product Content Styles

### PR DESCRIPTION
**Pull request by matthewroach**
## Description

* Change images in mock data to ones that have a background colour
* Update product content to allow styling of lists

## Jira Ticket

- [TBRANDS-114](https://arcpublishing.atlassian.net/browse/TBRANDS-114)

## Test Steps

1. Checkout this branch `git checkout TBRANDS-114-content-style-updates`
2. Use feature pack branch - `TBRANDS-114-content-style-updates`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/commerce-product-content-source-block,@wpmedia/product-content-block`
4. Create a new page
5. Add the product gallery block to the page
6. Set the global content to be
7. Content Source = commerce-product
8. SKU = `bounce-running-shoe` 
9. Validate the blue shoe pictures are loaded
10. Add product content block to the page
11. Select Product Details from the custom field Product Content
12. Uncheck the custom field collapsed on page load
13. Validate the lists are inline with the paragraphs

### Before

Microsoft Edge-2022-08-30-10-01 23

### After

Microsoft Edge-2022-08-30-10-01 33

 
## Dependencies or Side Effects

Feature Pack PR - https://github.com/WPMedia/arc-themes-feature-pack/pull/342

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/product-content-block/_index.scss [hunk 1]
```diff
@@ -1,5 +1,14 @@
 @use "@wpmedia/arc-themes-components/scss";
 
 .b-product-content {
+	ul {
+		@include scss.block-properties("product-content-ul");
```
Comment (nschubach) [inline blocks/product-content-block/_index.scss:5]: Feels a little weird here calling these by the tag name.  Maybe `product-content-items` and later `product-content-item`?

Comment (matthewroach) [reply blocks/product-content-block/_index.scss:5]: But it's not `items` we are allowing each tag to be controlled independently

Comment (JackHowa) [reply blocks/product-content-block/_index.scss:5]: would `product-content-list` work here as a compromise?

Comment (JackHowa) [reply blocks/product-content-block/_index.scss:5]: would `product-content-list` work here ?

Comment (nschubach) [reply blocks/product-content-block/_index.scss:5]: Oh, I see this is for styling the in-lined HTML... I suppose I'm be ok with `-list` and `-list-item`  It just feels dirty to me to use the tag names in the tokens.

Comment (matthewroach) [reply blocks/product-content-block/_index.scss:5]: Just realised we do the same in the article body block for HTML elements that are set directly

**Review by nschubach — APPROVED** [review]

**Pull request merged**